src/sentry/notifications/notifications/base.py

```diff
@@ -114,14 +199,24 @@ def get_project_link(self) -> str:
         project_link: str = absolute_uri(f"/{self.organization.slug}/{self.project.slug}/")
         return project_link
 
-    def record_notification_sent(self, recipient: Team | User, provider: ExternalProviders) -> None:
-        analytics.record(
-            f"integrations.{provider.name.lower()}.notification_sent",
-            actor_id=recipient.id,
-            category=self.get_category(),
-            organization_id=self.organization.id,
-            project_id=self.project.id,
-        )
-
     def get_log_params(self, recipient: Team | User) -> Mapping[str, Any]:
         return {"project_id": self.project.id, **super().get_log_params(recipient)}
+
+    def build_notification_footer(self, recipient: Team | User) -> str:
```

Comment:
Since we use `role_based_recipient_strategy` to generate the notification footer for request notifications, I wonder if we can make new strategy classes to generate this footer. But I'm not familiar enough with the notifications to know what that should look like.